src/process/rlimit.rs

```diff
@@ -35,6 +36,7 @@ pub fn getrlimit(resource: Resource) -> Rlimit {
 ///
 /// [POSIX]: https://pubs.opengroup.org/onlinepubs/9699919799/functions/setrlimit.html
 /// [Linux]: https://man7.org/linux/man-pages/man2/setrlimit.2.html
+#[cfg(not(target_arch = "loongarch64"))]
 #[inline]
 pub fn setrlimit(resource: Resource, new: Rlimit) -> io::Result<()> {
```

Comment:
Similarly, I assume loongarch64 should support the public `setrlimit` API even if there is no `__NR_setrlimit`, as it can use `prlimit64` instead.